lib/validator.js

```diff
@@ -86,7 +86,8 @@ function flatten(array) {
 }
 
 /**
- * @param {*} ast AST or array of ASTs
+ * @param {*[]} ast AST or array of ASTs
+ * @return {error[]} validation errors
```

Comment:
`*[]` is better than `*` but `error[]` is not since there is no type named `error`. Maybe also `*[]` for now?